backend/src/main/java/com/zpi/backend/game_instance/GameInstanceService.java

```diff
@@ -62,12 +66,13 @@ public GameInstanceDTO updateGameInstance(String uuid, UpdatedGameInstanceDTO up
     }
 
     //TODO Implementation of checking reservation, what about status?
-    public void deleteGameInstance(String uuid, Authentication authentication) throws GameInstanceDoesNotExistException, UserDoesNotExistException {
+    public void deleteGameInstance(String uuid, Authentication authentication) throws GameInstanceDoesNotExistException, UserDoesNotExistException, GameInstanceImageDoesNotExistException {
         User user = userService.getUser(authentication);
         Optional<GameInstance> gameInstanceOptional = gameInstanceRepository.findByUuidAndOwner(uuid, user);
+        //TODO Check if there is no reservation with status in progress or smth
         if (gameInstanceOptional.isEmpty())
             throw new GameInstanceDoesNotExistException("Game Instance (uuid = "+uuid+") does not exists or the User is not the Owner.");
-        gameInstanceImageService.deleteGameInstanceImagesByGameInstance(authentication, gameInstanceOptional.get().getUuid());
```

Comment:
Why is that changed? Here we delete images of the Game Instance, so we should use method ```deleteGameInstanceImagesByGameInstance``` not ```deleteGameInstanceImage```. The other method delete image by its (this image) uuid.

Reply:
idk merging develop->my branch was kinda broken and i had to do this 3 times, i'll change it